File: 백준/Silver/11004. K번째 수/K번째 수.js
let input = require('fs')
.readFileSync('/dev/stdin')
.toString().split('\n');

let n = Number(input[0].split(' ')[0]);
let k = Number(input[0].split(' ')[1]);

let array = input[1].split(' ').map(Number);

array.sort(
    function(a,b){
        return a - b
    }
)

console.log(array[k-1]);

File: 백준/Silver/11004. K번째 수/K번째 수.cs
using System;
using System.Collections.Generic;    
using System.Linq;

class Program
{
    static void Main()
    {
        int[] input = Console.ReadLine().Split(' ').Select(int.Parse).ToArray(); 
        int n = input[0];
        int k = input[1];
       
        List<int> array = Console.ReadLine().Split(' ').Select(int.Parse).ToList();
        
        array.Sort();
        
        Console.WriteLine(array[k-1]);

    }
}